.github/workflows/publish.yml

```
@@ -1,29 +1,23 @@
 name: Publish
-on: [push]
+on:
+  release:
+    types: [published]
 
 jobs:
   publish:
     runs-on: ubuntu-latest
-    if: startsWith(github.event.ref, 'refs/tags/')
 
     steps:
     - uses: actions/checkout@v2
 
-    - name: Set up Python 2
+    - name: Set up Python 3
       uses: actions/setup-python@v2
       with:
-        python-version: 2
-
-    - name: Check Tag
-      id: check-release-tag
-      run: |
-        if [[ ${{ github.event.ref }} =~ ^refs/tags/[0-9]+[.][0-9]+[.][0-9]+(rc[0-9]+|[.]dev[0-9]+)?$ ]]; then
-          echo ::set-output name=release_tag::true
-        fi
+        python-version: 3.9
```

3.9 may be safer, but I would not mind allowing future versions as well:

```suggestion
        python-version: 3
```